### ISSUE TITLE
Open Orca and others

### ISSUE THREAD

**Issue by shroominic** [shroominic/codeinterpreter-api#7]
Lookout for what the best open source model is to run LangChain Agents.
Implement this model and potentially others similar to #6 using ConversationalChat Agents.

**Comment by PedzacyKapec** [comment]
I agree.
Open Source Models like LLama 2 are the future.
LLama 2 already is surpassing chatgpt3.5 in most cases, apart from coding (for now)

**Comment by shroominic** [comment]
Yeah, coding is the important thing here. But I have some ideas for tricks on how to step by step guide the model through the process :D
